evcouplings/fold/haddock.py

```diff
@@ -0,0 +1,57 @@
+"""
+Functions for generating input to Haddock webserver
+
+Authors:
+  Anna G. Green
+"""
+
+
+def haddock_dist_restraint(resid_i, chain_i, resid_j, chain_j,
```

Comment:
OK, what I'm doing is the following:

1. made this function ambiguous so that it can also take in atom_i, atom_j parameters
2. Wrote a second distance restraints formatter that will write restraints files with atom types
3. Added restraint values for CA-CA restraints to the haddock yaml file

So currently the users would have to access the API to get these atom-specific restraint files. Do you think this is worth its own option in the protocol and config files?

Reply:
I think that API-only is fair enough for now - we can always easily add that option to the protocol later